### PR TITLE
Examine now shows if a head has had it's organs removed

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do
It adds a small quality of life examine to heads if they have been cut open.

## Why It's Good For The Game
Examining a full body without a brain will show the brain is missing, a decapitated head should be able to do the same thing.

## Images of changes

image

## Changelog
:cl:
tweak: Examine now shows if a head has had it's organs removed
/:cl:

### CODE REVIEW

**Comment by Inadvizable** [comment]
"disembowel" refers to the gastrointestinal tract specifically >:(

File: code/modules/surgery/organs/subtypes/standard.dm [hunk 1]
```diff
@@ -228,6 +228,11 @@
 	var/sec_facial_colour = "#000000"
 	var/f_style = "Shaved"
 
+/obj/item/organ/external/head/examine(mob/user)
+	. = ..()
+	if(!contents.len)
```
Comment (hal9000PR) [inline code/modules/surgery/organs/subtypes/standard.dm:233]:
```suggestion
	if(!length(contents))
```

**Comment by Contrabang** [comment]
> "disembowel" refers to the gastrointestinal tract specifically >:(

image
organ code

**Comment by Inadvizable** [comment]
> image 

ok certified :gnarg: moment right here

**Comment by Carthusia** [comment]
huzzah, no more "NURSE! FETCH ME A SPORK! I NEED TO CHECK FOR BRAAAINNNSSSS!"

**Comment by Sirryan2002** [comment]
This PR title is inaccurate, it will only display this message if all three organs- eyes, ear, and brain are removed. If there is no brain but there are eyes for example, the if check will fail.

**Comment by Contrabang** [comment]
> This PR title is inaccurate, it will only display this message if all three organs- eyes, ear, and brain are removed. If there is no brain but there are eyes for example, the if check will fail.

Good point, I'll change that. Even though, that should only be the time when this occurs is when all the organs are removed via cutting a decapitated head open.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**